.azure/pipelines/ci.yml

```diff
@@ -163,6 +147,7 @@ stages:
                 -all
                 -buildNative
                 -noBuildJava
+                -noRestore
```

Comment:
I'm least sure about using `-noRestore` in this build job because it's goes back and forth between X64 and X86 builds multiple times and the bitness can change how some projects are restored. Will kick off an internal build to make sure I didn't break anything. (Could add more `./restore.cmd` invocations if need be.)

Reply:
https://dev.azure.com/dnceng/internal/_build/results?buildId=599183

Reply:
Now https://dev.azure.com/dnceng/internal/_build/results?buildId=599363